ros_ws/src/autonomous/wallfollowing2/cfg/wallfollowing2.cfg

```diff
@@ -4,31 +4,31 @@ PACKAGE = "wallfollowing2"
 
 gen = ParameterGenerator()
 
-gen.add('min_throttle', double_t, 0, "TODO Tooltip", 0.2, 0, 1)
-gen.add('max_throttle', double_t, 0, "TODO Tooltip", 1.0, 0, 1)
+gen.add('min_throttle', double_t, 0, "Specifies the minimum speed.", 0.2, 0, 1)
+gen.add('max_throttle', double_t, 0, "Specifies the maximum speed.", 1.0, 0, 1)
 
-gen.add('radius_lower', double_t, 0, "TODO Tooltip", 2, 0.001)
-gen.add('radius_upper', double_t, 0, "TODO Tooltip", 30, 0.001)
+gen.add('radius_lower', double_t, 0, "Specifies a lower radius, used to calculate a speed limit.", 2, 0.001)
+gen.add('radius_upper', double_t, 0, "Specifies a upper radius, used to calculate a speed limit.", 30, 0.001)
 
-gen.add('steering_slow_down', double_t, 0, "TODO Tooltip", 4, 0)
-gen.add('steering_slow_down_dead_zone', double_t, 0, "TODO Tooltip", 0.2, 0)
+gen.add('steering_slow_down', double_t, 0, "Used to calculate the speed limit error.", 4, 0)
+gen.add('steering_slow_down_dead_zone', double_t, 0, "Used to calculate the speed limit error.", 0.2, 0)
 
-gen.add('high_speed_steering_limit', double_t, 0, "TODO Tooltip", 0.5, 0)
+gen.add('high_speed_steering_limit', double_t, 0, "Used to calculate the steering angle.", 0.5, 0)
 gen.add('high_speed_steering_limit_dead_zone',
-        double_t, 0, "TODO Tooltip", 0.2, 0)
+        double_t, 0, "Used to calculate the steering angle.", 0.2, 0)
 
-gen.add('max_acceleration', double_t, 0, "TODO Tooltip", 0.4, 0)
+gen.add('max_acceleration', double_t, 0, "Specifies the maximum spacceleration.", 0.4, 0)
 
-gen.add('corner_cutting', double_t, 0, "TODO Tooltip", 1.4, 0)
-gen.add('straight_smoothing', double_t, 0, "TODO Tooltip", 1.0, 0)
+gen.add('corner_cutting', double_t, 0, "Used to calculate the predicted car position.", 1.4, 0)
+gen.add('straight_smoothing', double_t, 0, "Used to calculate the predicted car position.", 1.0, 0)
 
-gen.add('barrier_size_realtive', double_t, 0, "TODO Tooltip", 0.1, 0)
-gen.add('barrier_lower_limit', double_t, 0, "TODO Tooltip", 1, 0)
-gen.add('barrier_upper_limit', double_t, 0, "TODO Tooltip", 15, 0)
-gen.add('barrier_exponent', double_t, 0, "TODO Tooltip", 1.4, 0)
+gen.add('barrier_size_realtive', double_t, 0, "Used to calculate the start and end point of the wall.", 0.1, 0)
+gen.add('barrier_lower_limit', double_t, 0, "Used to calulate the speed limit.", 1, 0)
+gen.add('barrier_upper_limit', double_t, 0, "Used to calulate the speed limit.", 15, 0)
+gen.add('barrier_exponent', double_t, 0, "Used to calulate the speed limit.", 1.4, 0)
```

Comment:
These should explain that the barrier is the distance that the car can drive straight. It is used to limit the speed at the end of a straight track part.